client/packages/programs/src/JsonForms/components/DateRange.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import { rankWith, ControlProps, uiTypeIs } from '@jsonforms/core';
+import { withJsonFormsControlProps } from '@jsonforms/react';
+import {
+  DateUtils,
+  useTranslation,
+  Box,
+  DetailInputWithLabelRow,
+  Typography,
+  LocaleKey,
+  DateTimePickerInput,
+} from '@openmsupply-client/common';
+import { z } from 'zod';
+
+import {
+  DefaultFormRowSx,
+  FORM_LABEL_WIDTH,
+  useZodOptionsValidation,
+} from '../common';
+import { PickersActionBarAction } from '@mui/x-date-pickers';
+
+const Options = z
+  .object({
+    hideClear: z.boolean().optional(),
+  })
+  .strict()
+  .optional();
+
+type Options = z.input<typeof Options>;
+
+export const dateRangeTester = rankWith(10, uiTypeIs('DateRange'));
+
+const UIComponent = (props: ControlProps) => {
```

Comment:
This one added for the encounter report - new JSON forms component designed to capture a date range (i.e. 2 inputs), "to" date shouldn't be less than "from" date - these should be captured together, rather than 2 separate date inputs